cfgov/v1/models/caching.py

```diff
@@ -83,3 +91,19 @@ def purge_all(self):
             )
         )
         resp.raise_for_status()
+
+
+@receiver(post_save, sender=Document)
+@receiver(post_save, sender=CFGOVRendition)
+def cloudfront_cache_invalidation(sender, instance, **kwargs):
+    media_base = settings.MEDIA_URL
+    if hasattr(settings, 'AWS_S3_CUSTOM_DOMAIN'):
+        media_base = settings.AWS_S3_CUSTOM_DOMAIN
+
+    url = urljoin(media_base, instance.url)
+
+    logger.info("Invalidating cache for " + url)
+
+    batch = PurgeBatch()
+    batch.add_url(url)
+    batch.purge(backends='cloudfront')
```

Comment:
Shouldn't `backends` be a list, per [this comment](https://github.com/wagtail/wagtail/blob/v1.13.4/wagtail/contrib/wagtailfrontendcache/utils.py#L145)?

Reply:
Listified in 570be79